### PR TITLE
making get_brawlers (issue #60)

### PR DESCRIPTION
**Pull request by 0dminnimda**
# Changes Description
1) all py files were refactored in pep8 style
(except for 97 lines in utils.py)
2) splitting BaseBox into BaseBox and BaseBoxList for convenience
(with the corresponding estates of both the models themselves and the models inheriting these models)
3) added:
- get_brawlers function
- model Brawlers
- BRAWLERS_URL in API
- nothing function to replace lambda expressions in typecasted
4) changed async.py and sync.py (now they are completely identical, in terms of meaning)
5) adding tests to files from tests /
6) tox for py38 now works
7) adding documentation to CHANGELOG.md, api.rst, index.rst
7.5) add some.py to gitignore

# Checklist
- [yes] Docstrings added ([NumpyDoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard))
- [added] If necessary, add to the documentation files
- [yes] Tox checked

### CODE REVIEW

**Comment by SharpBit** [comment]
Please revert everything and readd *only* the tests for `get_brawler`, the models/functions for `get_brawler` and the fix for tox for 3.8

- Please don't mess with code formatting, I purposely have the character limit as 120 chars since pep8's 80 character limit per line is just plain stupid and inconvenient
- You've added random f strings on docstrings where they're not needed and left in some Python-3.5-incompatible code (some fstrings). Again, please revert and only add the code needed for `get_brawler` and the changes to tox that lets you run it for Python 3.8

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+some.py
```
Comment (fourjr) [inline .gitignore:1]: i think extra files should not be added to the .gitignore

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
 All notable changes to this project will be documented in this file.
 
+
+## [Unnoun yet] - 6/27/20
+### Added
+- `get_brawlers` function to get available brawlers
+### Changed
+- splitting `BaseBox` into `BaseBox` and `BaseBoxList` for convenience
+### Fixed
+- tox for py38 works (not completely sure)
```
Comment (fourjr) [inline CHANGELOG.md:11]: tox should be tested first...

File: brawlstats/utils.py [hunk 3]
```diff
@@ -76,6 +78,12 @@ def get_datetime(timestamp: str, unix=True):
     else:
         return time
 
+
+# do nothing
+def nothing(value):
+    return value
```
Comment (fourjr) [inline brawlstats/utils.py:84]: i dont think its necessary to create a function here, the lambda works just fine and is cleaner

File: brawlstats/utils.py [hunk 4]
```diff
@@ -89,7 +97,7 @@ def wrapper(*args, **kwargs):
         for _, param in signature:
             converter = param.annotation
             if converter is inspect._empty:
-                converter = lambda a: a  # do nothing
+                converter = nothing
```
Comment (fourjr) [inline brawlstats/utils.py:100]: i dont think its necessary to create a function here, the lambda works just fine and is cleaner

**Comment by 0dminnimda** [comment]
I corrected everything, but you should know that pep8 standards did not come from the air!

File: brawlstats/utils.py (hunk 4 above)
Comment (0dminnimda) [reply brawlstats/utils.py:100]: lambda expression is created every time, and the function is defined, I think that it is worth avoiding lambda expressions, there is no problem to remove it

File: CHANGELOG.md (hunk 2 above)
Comment (0dminnimda) [reply CHANGELOG.md:11]: I do not often create tox files (so I'm not sure), but it works fine for me

File: brawlstats/utils.py (hunk 4 above)
Comment (0dminnimda) [reply brawlstats/utils.py:100]: lambda expression can result in unexpected performance issues

Comment (0dminnimda) [reply brawlstats/utils.py:100]: so I would recommend you leave this part alone

File: .gitignore (hunk 1 above)
Comment (0dminnimda) [reply .gitignore:1]: nothing bad here, I just need to ignore some files. it does not spoil, you can remove this change

File: CHANGELOG.md (hunk 2 above)
Comment (0dminnimda) [reply CHANGELOG.md:11]: here too that to the bottom post

File: .gitignore (hunk 1 above)
Comment (SharpBit) [reply .gitignore:1]: If you need another file, move it into another directory - do not add unnecessary stuff to the project's .gitignore file

Comment (0dminnimda) [reply .gitignore:1]: Ok

**Comment by SharpBit** [comment]
Transferred to PR #62

**Pull request closed without merging**